# Convergent evolution of the SARS-CoV-2 Omicron subvariants leading to the emergence of BQ.1.1 variant

## Summary
In late 2022, although the SARS-CoV-2 Omicron subvariants have highly diversified, some lineages have convergently acquired amino acid substitutions at five critical residues in the spike protein. Here, we illuminated the evolutionary rules underlying the convergent evolution of Omicron subvariants and the properties of one of the latest lineages of concern, BQ.1.1.


## Contents
* **Phylogenetic analysis:** a directory containing computational codes for phylogenetic analysis
* **Epidemic dynamics modeling analysis:** a directory containing computational codes for epidemic dynamics modeling analysis
* **resequencing_viral_genome:** a directory containing raw data (fastq files) for resequencing viral stock
* **GISAID acknowledgement table:** a directory containing GISAID acknowledgement tables describing the GISAID dataset used in this study.





## System requirements
* **Ubuntu** 20.04.4 LTS
* **R** v4.1.2
* **Python** v3.8.10
* **CmdStan** v2.30.1
* **CmdStanr (R library)** v0.5.3

System requirements specific to each script are denoted in the README in the corresponding subdirectory.

## Citation
Ito et al., **Convergent evolution of the SARS-CoV-2 Omicron subvariants leading to the emergence of BQ.1.1 variant** 2022 **bioRxiv**
doi: https://doi.org/10.1101/2022.12.05.519085


## License
This project is covered under the **Apache 2.0 License**.

